src/gui/screen_print_preview.cpp

```diff
@@ -134,10 +142,19 @@ GCodeInfoWithDescription::GCodeInfoWithDescription(window_frame_t *frame)
     }
 }
 
+static void print_button_press() {
+    if (!valid_printer_settings) {
+        MsgBoxTitle(_("WARNING:"), _("This G-CODE was set up for another printer type."),
+            Responses_Ok, 0, GuiDefaults::RectScreenBodyNoFoot);
+        Sound_Play(eSOUND_TYPE_SingleBeep);
```

Comment:
It seems to me, that the user is warned but they have no way to actually not-start the printing process. Right?

Reply:
Yes, that is the behaviour of MK3 in the same situation. Is something else requested?

Reply:
I think it might be nicer to give the user two options: „Ok“ and „Cancel“.